client/src/main/proto/spine/base/command.proto

```diff
@@ -95,7 +95,7 @@ message CommandContext {
 // Enumeration of possible failures when validating a command.
 enum CommandValidationError {
     // Reserved value.
-    UNKNOWN = 0;
+    UNDEFINED_COMMAND_VALIDATION_ERROR = 0;
```

Comment:
Please also note that zero is reserved by Protobuf for internal purposes. Presumably, it would show us that there was an error in enum transfer or the value wasn't set. So, I don't think we should use zero for the framework needs.